crates/web-sys/webidls/enabled/RTCRtpSender.webidl

```diff
@@ -51,28 +51,26 @@ dictionary RTCRtcpParameters {
   boolean   reducedSize;
 };
 
-dictionary RTCRtpCodecParameters {
-  unsigned short payloadType;
+dictionary RTCRtpCodec {
   DOMString      mimeType;
   unsigned long  clockRate;
```

Comment:
Apologies, I think these two have to be `required` after all.
You should take a look yourself and see what causes a breaking change in the API or not.